### PR TITLE
tests: fix basic20plus test for uc22 on rpi

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This test is failing because uc22 on rpi is not using grub, it uses piboot instead. 

The test is failing to find the kernel.efi

### CODE REVIEW

File: tests/core/basic20plus/task.yaml
Comment (ZeyadYasser) [inline tests/core/basic20plus/task.yaml:1]: It seems that the grub check is skipped successfully but it seems to be failing fail to find the kernel image on core-22-arm: https://github.com/snapcore/snapd/actions/runs/5810065891/job/15751039942?pr=13074#step:8:761
https://github.com/snapcore/snapd/blob/0cb8e9d29b98c3108adaaf0b2f08a90136cc032b/tests/core/basic20plus/task.yaml#L57-L59

**Comment by alfonsosanchezbeato** [comment]
The description is a bit confusing, afaiu this is testing pc-kernel for arm64, which does use grub and has kernel.efi.

**Comment by sergiocazzolato** [comment]
@alfonsosanchezbeato @ZeyadYasser I updated the title and description. The issue is on rpi.

File: tests/core/basic20plus/task.yaml
Comment (ZeyadYasser) [reply tests/core/basic20plus/task.yaml:1]: @sergiocazzolato any idea why this happens?

**Review by ZeyadYasser — APPROVED** [review]
LGTM, Thank you!

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM

**Pull request merged**